crates/nu-command/src/network/http/client.rs

```diff
@@ -561,14 +560,12 @@ fn transform_response_using_content_type(
     resp: Response,
     content_type: &str,
 ) -> Result<PipelineData, ShellError> {
-    let regex = Regex::new("\"").expect("Failed to create regex");
-    let content_type_trim = regex.replace_all(content_type, "").to_string();
-
-    let content_type = mime::Mime::from_str(&content_type_trim).map_err(|err| {
-        LabeledError::new(err.to_string())
-            .with_help("given unknown MIME type, or error parsing MIME type")
-            .with_label(format!("MIME type unknown: {content_type_trim}"), span)
-    })?;
+    let content_type = mime::Mime::from_str(content_type)
+        // there are invalid content types in the wild, so we try to recover
+        // Example: `Content-Type: "text/plain"; charset="utf8"` (note the quotes)
+        .or_else(|_| mime::Mime::from_str(&content_type.replace('"', "")))
```

Comment:
I thought this wouldn't be enough, which is why I was doing a replace_all with regex but it seems like `"text/xml; charset=utf8"` still parses well. So, good move to remove regex.